pyproject.toml

```diff
@@ -142,7 +143,10 @@ myst-parser = "*"
 [tool.poetry.group.tests.dependencies]
 pytest = "^7.4.0"
 pytest-subtests = "^0.11.0"
-numpy = ">=1.24.3"
+numpy = [
+  { "version" = ">=1.24.3", "python" = "<3.12" },
```

Comment:
Shouldn't this be `{ "version" = "~1.24.3", "python" = "<3.12" },`? (to express < 1.25.0)

Reply:
Poetry takes care of selecting a version of `numpy` that is compatible with the rest of the dependencies. Therefore [for Python 3.8, it updates only up to `1.24.4`](https://github.com/linkml/linkml/blob/add-support-for-python3.12/poetry.lock#L1596) and not further up.
The tests will tell us, if I'm right or not 😄

Reply:
Confirmed, resolving this conversation.

Still stumbling upon a different issue...